packages/@o3r/test-helpers/src/prepare-test-env.ts

```diff
@@ -43,14 +47,15 @@ export async function prepareTestEnv(folderName: string, options?: PrepareTestEn
   const logger = options?.logger || console;
   const yarnVersionParam = options?.yarnVersion;
   const rootFolderPath = process.cwd();
-  const itTestsFolderPath = path.resolve(rootFolderPath, '..', 'it-tests');
+  const itTestsFolderPath = path.resolve(rootFolderPath, '..', 'it-tests', options?.packageScope || '');
```

Comment:
not linked to this PR but it would be good to keep the `it-tests` folder inside the repository in a .tmp folder
so it can be properly be cleaned-up with git clean

Reply:
we can put it in a **tmp** folder which is already gitignored.

Reply:
unfortunate :(